README.md

```diff
@@ -832,15 +832,15 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 6.8. Avoid using the Node.js crypto library for handling passwords, use Bcrypt
+## ![✔] 6.8. Protect Users' Passwords/Secrets
```

Comment:
This part looks good to me

Reply:
@josh-hemphill Consider This sounds a bit vanilla, 'of course I should protect users password, but explain how'. Consider including the punch: 'Protect Users' Passwords/Secrets using BCrypt or Script'.